bazel/external/proxy_wasm_cpp_host.BUILD

```diff
@@ -0,0 +1,54 @@
+licenses(["notice"])  # Apache 2
+
+package(default_visibility = ["//visibility:public"])
+
+cc_library(
+    name = "include",
```

Comment:
Why we use this ambiguous name? I think that `name = "proxy_wasm_interface"` is better.